crates/turbo-malloc/src/counter.rs

```diff
@@ -0,0 +1,72 @@
+use std::{
+    cell::UnsafeCell,
+    ptr::NonNull,
+    sync::atomic::{AtomicUsize, Ordering},
+};
+
+pub static ALLOCATED: AtomicUsize = AtomicUsize::new(0);
```

Comment:
This probably shouldn't be publicly accessible, since we don't want external items to be able to `fetch_update` this. Make a `get` fn instead?